index.js

```diff
@@ -1,5 +1,5 @@
 import {ApolloServer} from 'apollo-server';
-import {resolvers, typeDefs} from './schema.js';
+import {resolvers, typeDefs} from './schema.mjs';
```

Comment:
I don't understand why I had to do this change, but eslint was complaining. Guess it wasn't complaning before, but any idea how this worked before and stopped working with my PR?

Reply:
I wonder if it has anything to do with the version of node you're running. I'll try to run it on my machine and see if it's still unhappy.